### PR TITLE
Update text

### PR DESCRIPTION
**Pull request by TaiWilkin**
## Overview

Updates text copy across site to match [requested changes](requested changes).

Updates the questions accordion to say "options" instead of "questions", and removes 2020-specific link on the questions selection page.

Hides currently selected geographies from the list of currently selected geographies if they are unavailable for all currently selected years. 

Connects #115 

### Demo

Screen Shot 2021-11-16 at 8 44 51 AM
Screen Shot 2021-11-16 at 8 45 04 AM
Screen Shot 2021-11-16 at 8 46 01 AM
Screen Shot 2021-11-16 at 8 46 20 AM
Screen Shot 2021-11-16 at 8 46 31 AM

## Testing Instructions

 * In the Netlify preview, compare the text in the homepage header, the about modal, and the FAQ modal, to [the text provided](https://docs.google.com/document/d/1l8bZNPxtpatzj4UoO83mhdmcAY4qiDgPD0yaT5TNoKU/edit#heading=h.c8dzwehzyqmz).
 * On the questions page, confirm that the accordion text says 'options' instead of questions, and that the link above the accordion has been removed. 
 * Select a country which is available for all years and one which is only available for one year. On the years page, select the unavailable year, then click next. On the questions page and the charts page, the unavailable geography should be hidden. 
 * On the years page, select the available year and then click next. On the questions page and the charts page, both geographies should be shown.

### CODE REVIEW

**Comment by rajadain** [comment]
Taking a look now

File: src/app/src/components/AboutModal.jsx [hunk 1]
```diff
@@ -40,49 +40,32 @@ const AboutModal = ({ isOpen, onClose }) => {
...
-                                were surveyed on Facebook to capture household
-                                gender dynamics during the COVID-19 pandemic.
-                                The data provides a global snapshot to help
-                                researchers, NGOs, global development
-                                institutions, and gender equality advocates
-                                identify and track trends and progress towards
-                                the UN Sustainable Development Goals (
+                                The survey has been fielded in 2020 and 2021
+                                over 200 countries. Public, aggregate datasets
+                                for the 2020 and 2021 surveys at the{' '}
```
Comment (rajadain) [inline src/app/src/components/AboutModal.jsx:57]:
```suggestion
                                for the 2020 and 2021 surveys are available at the{' '}
```

File: src/app/src/components/AboutModal.jsx [hunk 2]
```diff
@@ -40,49 +40,32 @@ const AboutModal = ({ isOpen, onClose }) => {
...
+                                , and de-identified, record-level response data
+                                to most questions may be available to research
+                                and nonprofit organizations for research
+                                purposes. Meta only shares data publicly once it
+                                is aggregated to the country or region level.
+                                Before sharing with Data for Good partners, Meta
+                                ensures that all survey data are de-identified.
                             </StyledText>
                         </StyledAccordionItem>
                         <StyledAccordionItem title='How the survey contributes to Facebook’s mission and the UN Sustainable Development Goals'>
```
Comment (rajadain) [inline src/app/src/components/AboutModal.jsx:71]: Should Facebook be switched to Meta here?

File: src/app/src/components/Visualizations.jsx [hunk 3]
```diff
@@ -54,6 +54,21 @@ const Visualizations = () => {
         }
     }, [showSurvey, surveyHasBeenDisplayed, dispatch]);
 
+    const config = CONFIG[geoMode];
```
Comment (rajadain) [inline src/app/src/components/Visualizations.jsx:57]: Will this crash if `geoMode` is `null` or `undefined`?

**Review by rajadain — APPROVED** [review]
+1 tested, edits look good, match the content sent from the clients. Found some small issues, may be optionally fixed, but overall this is good.

File: src/app/src/components/AboutModal.jsx (hunk 1 above)
Comment (TaiWilkin) [reply src/app/src/components/AboutModal.jsx:57]: Confirmed with Samhir

File: src/app/src/components/AboutModal.jsx (hunk 2 above)
Comment (TaiWilkin) [reply src/app/src/components/AboutModal.jsx:71]: Confirmed with Samhir

File: src/app/src/components/Visualizations.jsx (hunk 3 above)
Comment (TaiWilkin) [reply src/app/src/components/Visualizations.jsx:57]: In a number of locations the app will crash if `geoMode` is `null` or `undefined`. It is set when the site loads and should never be `null`, but I'm going to add a follow-up card to generally add some protections around this.

**Comment by TaiWilkin** [comment]
Thank you for reviewing!

**Pull request merged**